src/app/(profile)/(components)/ContactForm/ContactForm.tsx

```diff
@@ -31,19 +32,22 @@ export default function ContactForm({
 }) {
   const { runAsync, loading } = useAsyncAction()
   const { addToast } = useToast()
+  const { refreshProfilesWithContactRequests } = useProfiles()
```

Comment:
**nit:** no need to refresh them - we can just map changed ProfieModel in our global state without refetching